### DISCUSSION TITLE
Any examples of this used outside of Next?

### DISCUSSION THREAD

**Discussion by ghost** [shadcn-ui/ui#559] [Q&A]
Are there any examples of this being used outside of Next, particularly in React?

**Comment by sankaSanjeeva** [comment]
@murrayobrien I created a Vite project(React typescript) and followed the manual installation method.
Additionally, I updated `tsconfig.json` and `vite.config.ts` to work `@` alias properly
```
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@/*": ["src/*"]
    }
  }
}
```
![Screenshot 2023-06-09 at 08 39 31](https://github.com/shadcn/ui/assets/44424604/c5ff67a4-2828-403e-918c-ceb37d417f1b)

**Comment by pratikrath126** [comment]
Yes, shadcn/ui works with any React framework (Vite, Remix, etc.) since it’s just copy-paste components using Radix + Tailwind. You can check the manual installation guide in the docs: https://ui.shadcn.com/docs/installation/manual or try the Vite guide: https://ui.shadcn.com/docs/installation/vite. It’s fully compatible outside of Next.js!

**Comment by eruens** [comment]
Yes, shadcn/ui works with Vite, Remix, Astro, and Laravel. The CLI auto-detects your framework. For Vite run: npx shadcn@latest init. For Remix: npx shadcn@latest init. For Astro, first add React: npx astro add react, then: npx shadcn@latest init. Full install guide at https://ui.shadcn.com/docs/installation/vite
